cmd/rosa/main.go

```diff
@@ -105,3 +113,27 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func versionCheck() {
+	rprtr := reporter.CreateReporterOrExit()
+
+	currVersion, err := gversion.NewVersion(info.Version)
+	if err != nil {
+		_ = rprtr.Errorf("There was a problem verifying the current version of rosa. Please be aware that you are possibly running an outdated version: %w", err)
+		os.Exit(1)
```

Comment:
Some other thoughts I had here - instead of `os.Exit(1)` here, would we rather either of the following?

a.) Immediately prompt to continue and skip the mirror check
b.) Continue without any further prompts

Reply:
I would lean towards option `b` as if there is an error with the check I dont think we would want to block. I would push to just warn the user that we could not verify and there is the risk they have an outdated version. To manually check a user can run `rosa verify rosa` or `rosa version`